tools/test_containers_compose

```diff
@@ -20,7 +20,7 @@ wait_until() {
 
 setup_containers() {
   for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done
-  sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d
+  sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d || docker-compose logs
```

Comment:
I assume that `docker-compose logs` would return 0 as exit code so the failure of the previous command would not abort the script execution

Reply:
mm yes, you're correct

Reply:
Fixed